forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -83,13 +84,9 @@ class ForageSDK : ForageSDKInterface {
 
         // TODO: replace Log.getInstance() with Log() in future PR
         val logger = Log.getInstance()
-        logger.i(
-            "[HTTP] Tokenizing Payment Method",
-            attributes = mapOf(
-                "merchant_ref" to merchantId,
-                "customer_id" to customerId
-            )
-        )
+            .addAttribute("merchant_ref", merchantId)
+            .addAttribute("customer_id", customerId)
+        logger.i("[Android SDK] Tokenizing Payment Method")
```

Comment:
Why are we changing the prefix? We already know this log is from the Android SDK because it's logged under the `android-sdk` service.

Reply:
`HTTP` felt misleading given that the logs were coming from `ForageSDK`. I will update to `ForageSDK` to be inline with iOS log format to make it clearer that the logs are originating from the entrypoint of the SDK

Reply:
0881b1a0a380b091cf28d9474ff4839f51cc4f2c